lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -1435,6 +1441,7 @@ def get_car_api(self, url, checkReady=True, provesOnline=True):
                 elif req.status_code == 429:
                     # We're explicitly being told to back off
                     self.errorCount = max(30, self.errorCount)
+                    self.carapi.updateCarApiLastErrorTime()
```

Comment:
I'm sorry, but unfortunately it's still wrong: The vehicle parameter is missing.

It should be:

self.carapi.updateCarApiLastErrorTime(**self**)

Just the same as a few lines later:

https://github.com/ngardiner/TWCManager/blob/b4d1536cfccf30f9b901cb2e471eebbe1f4ff594/lib/TWCManager/Vehicle/TeslaAPI.py#L1466

Reply:
No problem at all. I did notice the `(self)` before but did not understand it. Makes sense now that you explained we are in the `CarApiVehicle` class. Fixed it.